### PR TITLE
Publish on pypi

### LINKED ISSUE #14: New release needed
@sebastienbeau can you do a new release?

Also, please have a look at pending PRs :pray:

### PR DESCRIPTION
**Pull request by simahawk**
Closes #14

### CODE REVIEW

**Comment by simahawk** [comment]
we should get rid of travis tests too :smile: 

Ref: https://github.com/OCA/odoo-test-helper/issues/16

File: setup.py [hunk 1]
```diff
@@ -16,25 +16,28 @@
...
-    url="http://github.com/akretion/odoo-test-helper",
+    author="Odoo Community Association (OCA)",
+    author_email="support@odoo-community.org",
+    url="http://github.com/OCA/odoo-test-helper",
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Intended Audience :: Developers",
-        "License :: OSI Approved :: "
-        "GNU Affero General Public License v3 or later (AGPLv3+)",
+        "License :: OSI Approved :: GNU Affero General Public License v3",
```
Comment (sbidoul) [inline setup.py]: Do we want to use LGPL here so we can use it in LGPL addons ?

File: CHANGES.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Changes
 .. - ...
 
 
-0.0.1
---------
+1.0.0
```
Comment (sbidoul) [inline CHANGES.rst:22]: There is already 1.1.0 on PyPI

**Review by sbidoul — APPROVED** [review]
LGTM

File: setup.py (hunk 1 above)
Comment (simahawk) [reply setup.py]: oh, good point. Fine for me. @sebastienbeau @lmignon I assume is fine for you both :wink:

Comment (simahawk) [reply setup.py]: @sbidoul OTOH technically this is not a dependency to ship a module.
BTW we should get back to talk about adding `tests_require` key in the odoo manifest :wink:

File: CHANGES.rst (hunk 2 above)
Comment (simahawk) [reply CHANGES.rst:22]: Alright, I switch to 2.0.0 as we change license.

File: setup.py (hunk 1 above)
Comment (sbidoul) [reply setup.py]: > BTW we should get back to talk about adding tests_require key in the odoo manifest

Yep. It should not be particularly difficult:

1. decide on the key - python-test in external_dependencies ?
2. use it:
2.a. update oca/oca-ci to use `manifestoo list-external-dependencies` for that key like it already does for `deb` dependencies
2b. better: update setuptools-odoo to add a `test` extra with these and update oca/oca-ci to do `pip install -e ./setup/theaddon[test]`
3. organise the transition, teach, ...

Comment (simahawk) [reply setup.py]: where should we move this discussion? `setuptools-odoo`? I'll open an RFC.

**Pull request merged**